tests/test/Revert.t.sol

```diff
@@ -37,4 +43,21 @@ contract CTest {
         assert(!result);
         assert(c.num() != x);
     }
+
+    function check_RevertBalance(bool paused, uint256 amount) public {
```

Comment:
cool! do we also have a check to verify that `code` is properly reverted too?

Reply:
ah I see https://github.com/a16z/halmos/issues/145 👀

Reply:
#145 is a different one.  i added a test for reverting code info.